skactiveml/regressor/tests/test_wrapper.py

```diff
@@ -284,5 +288,19 @@ def test_predict(self):
         np.testing.assert_array_equal(np.ones(3), std_pred)
 
     def test_partial_fit(self):
-        # TODO: add Test
-        pass
+        X_all, y_all = make_regression(n_samples=300, random_state=0)
+        X_fit, y_fit = X_all[:200], y_all[:200]
+        X_new, y_new = X_all[200:], y_all[200:]
+
+        class GaussianProcessRegressorDummy(GaussianProcessRegressor):
+            def partial_fit(self, X, y):
+                return self.fit(X, y)
```

Comment:
https://scikit-learn.org/0.15/modules/generated/sklearn.linear_model.SGDRegressor.html#sklearn.linear_model.SGDRegressor
We could use the SGDRegressor instead of a dummy class